src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -273,6 +276,23 @@ class ProjectGeneralSettings extends AsyncView {
             fields={[fields.resolveAge]}
           />
 
+          {jsonFormProps.features.has('set-grouping-config') && (
```

Comment:
I would move this logic into the form data declaration because search potentially uses that data too e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx#L210